SingularityUI/app/components/machines/usage/SlaveResourceHealthMenuItems.js

```diff
@@ -56,6 +56,7 @@ const maybeLink = (name, value) => {
 };
 
 const SlaveResourceHealthMenuItems = ({stats}) => {
+  stats = _.filter(stats, (stat) => _.values(STAT_NAMES).includes(stat.name));
```

Comment:
If you're going to use `includes` (ES2016), I'd suggest that you don't use `_.contains` elsewhere in your code. You can also directly use `Array.prototype.filter`